src/common/utils/recommend.ts

```diff
@@ -210,18 +214,17 @@ function isBetweenDates(point: Column, dateFrom: Date, dateTo: Date) {
  * Generates location+metric-specific section of intro blurb
  */
 export function getDynamicIntroCopy(
-  locationName: string,
+  projection: Projection,
   metricValues: { [metric in Metric]: number | null },
 ): string {
+  const { locationName } = projection;
   const hasPosTestRate = isNumber(metricValues[Metric.POSITIVE_TESTS]);
 
-  const blurb = `Based on ${locationName}'s ${getMetricNameForCompare(
-    Metric.CASE_DENSITY,
-  ).toLowerCase()} (${formatValue(
-    Metric.CASE_DENSITY,
-    metricValues[Metric.CASE_DENSITY],
-    '',
-  )})${
+  const casesPerWeekMetricName = 'new cases per 100k in the last 7 days';
```

Comment:
`casesPerWeekMetricName` needs to also be applied to `getSharedLocationMetricCopy` (the location-specific text in the modal)